README.md

```diff
@@ -6,6 +6,7 @@ The Helm chart uses the [1Password/op-scim-helm](https://github.com/1Password/op
 on the Google Cloud Platform.
 
 To work with this deployment bundle, you will need to have the following tools installed:
+
```

Comment:
Some of added line changes are intentional? 🤔

Reply:
Auto linted 😄